notepad/index.js

```diff
@@ -6,6 +6,7 @@ const toolbarOptions = [
   ['blockquote', 'code-block'],
 
   [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
+  [{ 'size': ['small', false, 'large', 'huge'] }],  // custom dropdown
```

Comment:
The default text in the header and size dropdowns appears to be the same ("normal") meaning unless you memorize order you don't know what you'll be controlling until you click. Any way to do better? No big deal if there's nothing easy to do here.